### ISSUE TITLE
Multiple machine keys

### ISSUE THREAD

**Issue by shawwn** [Tarsnap/tarsnap-gui#205]
I'm not sure there's a way to view multiple machines, and as a user I feel nervous changing the machine key. ("Would that remove all existing scheduled jobs or have destructive effects?")

I figured it might be useful to open an issue for it, but it's probably not a pressing feature.

**Comment by shawwn** [comment]
To clarify the scenario a bit: I have a tarsnap key in my Dropbox from many years ago. I know that there are a few archives associated with it, and I thought it would be interesting to view them using Tarsnap GUI. But I wasn't sure if this was supported or not.

I imagine that might be a pretty common scenario for new users, but that's just speculation.

**Comment by gperciva** [comment]
Right.  The app theoretically includes an `--appdata DIR` command-line argument, but it doesn't work fully (I have most of a fix ready).

... I just remembered that most people don't start gui apps on the command-line.  Ok, this will take a bit more thought.
